src/utils/url.ts

```diff
@@ -40,7 +40,7 @@ export function normalizeSchemaUrl(url: string): string {
 
 function normalizeUrl(url: string): string {
   let fullUrl: string;
-  if (/^[^:]+:\/\/./.test(url)) {
+  if (/^[^:]+:[\/]*./.test(url)) {
```

Comment:
Nit: consider the below to disallow: `http://////`
```suggestion
  if (/^[^:]+:(\/{2})?[^\/].*/.test(url)) {
```